zerver/openapi/zulip.yaml

```
@@ -6551,14 +6575,43 @@ paths:
                     </tr>
                     <tr>
                         <td><code>wildcard_mentioned</code></td>
+                        <td>
+                            Whether this message contained either
+                            <a href="/help/mention-a-user-or-group#mention-everyone-on-a-stream">stream wildcard mention</a>
+                            like @**all** or topic wildcard mention (@**topic**). Cannot be changed
+                            by the user directly, but can change if the message is edited to add/remove
+                            a stream or/and topic wildcard mention.
+
+                            **Changes**: In Zulip 8.0 (feature level 224), the `wildcard_mentioned` flag
+                            was deprecated in favor of the `stream_wildcard_mentioned` and `topic_wildcard_mentioned`
+                            flags. The `wildcard_mentioned` flag exists for backwards compatibility with
+                            older clients and equals `stream_wildcard_mentioned` || `topic_wildcard_mentioned`.
+                            Clients interacting with older servers should treat this field as an old name
+                            for `stream_wildcard_mentioned`. @topic mentions were not available prior to
+                            this feature level.
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><code>stream_wildcard_mentioned</code></td>
                         <td>
                             Whether this message contained
-                            <a href="/help/mention-a-user-or-group#mention-everyone-on-a-stream">wildcard mention</a>
+                            <a href="/help/mention-a-user-or-group#mention-everyone-on-a-stream">stream wildcard mention</a>
                             like @**all**. Cannot be changed by the user directly, but
                             can change if the message is edited to add/remove
-                            a wildcard mention.
+                            a stream wildcard mention.
+
+                            **Changes**: New in Zulip 8.0 (feature level 224).
                         </td>
                     </tr>
+                    <tr>
+                        <td><code>topic_wildcard_mentioned</code></td>
+                        <td>
+                            Whether this message contained topic wildcard mention (@**topic**).
```

```suggestion
                            Whether this message contained a topic wildcard mention (@**topic**).
```
Looks like we don't have a documentation we can link to yet, which I assume is a follow-up task.

---

Yes